plugins/msg-broker/mcollective/lib/openshift/mcollective_application_container_proxy.rb

```diff
@@ -2029,9 +2029,26 @@ def move_gear(gear, destination_container, destination_district_uuid, change_dis
           end
         end
       else
-        # leave quota the same if node profile does not change
-        quota_blocks = Integer(gear_quota[3])
-        quota_files = Integer(gear_quota[6])
+        # if gear is over disk quota limit, increase destination quota by 1% so gear move, start, stop, restart complete
+        # get the base quota blocks and files destination container
+        destination_quota = destination_container.rpc_get_facts_direct(["quota_blocks", "quota_files"])
+        # set destination container quotas
+        quota_blocks = Integer(destination_quota[:quota_blocks])
+        quota_files = Integer(destination_quota[:quota_files])
+        # get current block and file usage from the source container
+        blocks_used = Integer(gear_quota[1])
+        if blocks_used >= quota_blocks
```

Comment:
I think we probably need some tolerance here.  It can probably be something like 0.9999, but there are definitely case where usage is a few kilobytes under quota and things still fail due to a combination of rewriting the state file and writing logs.